Sparklines

jordansread opened the pull request:
![image](https://cloud.githubusercontent.com/assets/2349007/19216047/487c8e36-8d74-11e6-898d-0fda21eadef6.png)

probably don't want to do this, but wanted to see if this worked better than coloring the dots.

Commit:
sparklines

Commit:
dot hover and other css

Commit:
conflict

Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew into sparklines

# Conflicts:
#	viz.yaml

Commit:
reverting so it works

Commit:
fixing viz that got lost

Commit:
library needed

Commit:
sparklines on the side!

Commit:
sparkling styling

Commit:
sparkline masking

Commit:
animated sparks

Commit:
with masking transparency

Commit:
merge w/ play button

Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew into sparklines

# Conflicts:
#	scripts/process/process.classifyBins.R
#	viz.yaml

Commit:
button?

jread-usgs commented:
This PR now has sparklines on the side and a subsets of gages after pruning to those within a specified radius of the track:

![image](https://cloud.githubusercontent.com/assets/2349007/19217995/e72fff58-8db1-11e6-8d70-5676989ebcd6.png)

They draw w/ the animation of the track

Commit:
minor style things

Commit:
style

Commit:
Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew into sparklines

Commit:
trying to fix yaml

Commit:
tool tips

Commit:
merge conflicts

Merge branch 'master' of github.com:USGS-VIZLAB/hurricane-matthew into sparklines

# Conflicts:
#	viz.yaml

Commit:
clean up indents
